src/patternfly/components/Table/table-grid.scss

```diff
@@ -119,7 +119,6 @@
   --#{$table}--m-grid__check--favorite--MarginInlineStart: var(--pf-t--global--spacer--xl);
 
   // * Table grid action
-  --#{$table}--m-grid__action--MarginBlockStart: #{pf-size-prem(6px)};
```

Comment:
This doesn't look necessary:

Before
<img width="505" alt="Screenshot 2024-07-03 at 3 57 41 PM" src="https://github.com/patternfly/patternfly/assets/35148959/092162bf-ac39-4fed-b6c7-605aa24e8a6e">

After. The cell's top padding matches the button's, so no margin on the action wrapper aligns the button text with the cell text
<img width="513" alt="Screenshot 2024-07-03 at 3 57 58 PM" src="https://github.com/patternfly/patternfly/assets/35148959/71a51eda-9eb6-4efc-a75e-69c4657182f0">